Revoke-Obfuscation.psm1

```diff
@@ -1178,6 +1185,31 @@ http://www.leeholmes.com/blog/
                 }
             }
         }
+        "Helix" {
+            # Use API to pull eventid=4104.
+            # Defeat the 1k event limitation using the "stream"="true" option in your request.
+            # Stream returns raw JSON which needs to be split by "`n" and convertfrom-json before providing to this function
+            
+            # Parse out scriptblock text ID#
+            $Helix.msg -match "\((\d+)\sof\s(\d+)\)"|Out-Null
```

Comment:
This should probably be:
```
if($Helix.msg -match (...))
{
   ...
}
```

Reply:
What does a Helix message look like when extracting from a non en-US system? The word "of" won't be there in the PowerShell logs.